traefik/templates/_podtemplate.tpl

```diff
@@ -27,6 +27,10 @@
       {{- with .Values.deployment.dnsPolicy }}
       dnsPolicy: {{ . }}
       {{- end }}
+      {{- with .Values.deployment.dnsConfig }}
+      dnsConfig:
```

Comment:
Instead of allowing plain YAML, could you check individual fields as @eddykaya did [here](https://github.com/traefik/traefik-helm-chart/pull/815) ?

Reply:
yes i can, At level one i suppose ?

```
.Values.deployment.dnsConfig.nameservers
.Values.deployment.dnsConfig.searches
.Values.deployment.dnsConfig.options
```

Reply:
I try something https://github.com/traefik/traefik-helm-chart/commit/4a71ed1c3e96e9d44dba31eaf5d7ea2dfd368597 does is it what you were thinking ?

Reply:
Yes perfect :+1: